empress/support_files/js/animation-panel-handler.js

```diff
@@ -101,6 +99,29 @@ define(["Colorer", "util"], function (Colorer, util) {
         this.trajectory.disabled = disableStatus;
     };
 
+
+    /**
+     * Enable/disable all the UI elements in the panel.
+     *
+     * @param{Bool} enabled Whether controls should be enabled (true) or
+     *                      disabled (false).
+     */
+    AnimationPanel.prototype.setEnabled = function (enabled) {
+        this.colorSelect.disabled = !enabled;
+        this.gradient.disabled = !enabled;
+        this.trajectory.disabled = !enabled;
+
+        this.collapseChk.disabled = !enabled;
+        this.lWidth.disabled = !enabled;
+
+        this.pauseBtn.disabled = !enabled;
+        this.startBtn.disabled = !enabled;
+        this.stopBtn.disabled = !enabled;
+        this.resumeBtn.disabled = !enabled;
+        this.prevFrameBtn.disabled = !enabled;
+        this.nextFrameBtn.disabled = !enabled;
```

Comment:
I doubt that doing a bunch of negations will be a problem, but may as well try to make fast :)

```suggestion
        var isDisabled = !enabled;
        this.colorSelect.disabled = isDisabled;
        this.gradient.disabled = isDisabled;
        this.trajectory.disabled = isDisabled;

        this.collapseChk.disabled = isDisabled;
        this.lWidth.disabled = isDisabled;

        this.pauseBtn.disabled = isDisabled;
        this.startBtn.disabled = isDisabled;
        this.stopBtn.disabled = isDisabled;
        this.resumeBtn.disabled = isDisabled;
        this.prevFrameBtn.disabled = isDisabled;
        this.nextFrameBtn.disabled = isDisabled;
```